### PR TITLE
reading from "test" file

### PR DESCRIPTION
**Pull request by kostana1**
Please review Test class and quiz package.
Test class was created only for testing purposes. Will be put in right package as soon as the code inside works properly. The code inside reads everything from "test" file except for line "Never,0". Any advises why this happens will be gladly accepted

### CODE REVIEW

File: src/main/java/com/quiz/Question.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        for(int i=0; i < this.questionAnswers.size(); i++) {
+            Answer existedAnswer = this.questionAnswers.get(i);
+            if(existedAnswer.getAnswerDescription().equals(answerDescription)) {
+                return existedAnswer;
+            }
+        }
+        return null;
+    }
+
+    public boolean addAnswer(String answerDescription, int points) {
```
Comment (martojdk) [inline src/main/java/com/quiz/Question.java:26]: pass the whole object, not only his fields

File: src/main/java/com/service/Test.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                while ((fileData = bufferedReader.readLine()) != null && !fileData.isEmpty()) {
+                    String[] answerData = fileData.split(",");
+                    String answerDescription = answerData[0];
+                    int answerPoint = Integer.parseInt(answerData[1]);
+                    counterAnswerLines++;
+
+                    if(counterAnswerLines == 4) {
+                        break;
+                    }
+                    question.addAnswer(answerDescription, answerPoint);
```
Comment (martojdk) [inline src/main/java/com/service/Test.java:55]: is the last one added ? you seem to break out from the cycle before doing so

File: src/main/java/com/service/Test.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                }
+
+                for(int j=0; j < question.questionAnswers.size(); j++) {
```
Comment (martojdk) [inline src/main/java/com/service/Test.java:58]: what is this 😄

File: src/main/java/com/service/CreatePersonService.java [hunk 4]
```diff
@@ -20,6 +21,12 @@
     public static final String PATTERN_REGEXP_YEAR = "\\d{4}-[01]\\d-[0-3]\\d";
     public static final String USE_INTEGERS_ONLY = "Use integers as per description";
 
+    public static final String FIRST_QUESTION = "Select a number from 1 to 10 to assess your fairness when answering ";
+    public static final String SECOND_QUESTION = "How often do you drink? " + "\n1 - I don't drink; 2 - Not very often; 3 - Every other day; 4 - Every day";
```
Comment (martojdk) [inline src/main/java/com/service/CreatePersonService.java:25]: remove +

File: src/main/java/com/person/Person.java [hunk 5]
```diff
@@ -51,14 +53,49 @@ public UUID getUuid() {
         return uuid;
     }
 
+    public int getQuestion() {
+        return question;
+    }
+
+<<<<<<< HEAD
```
Comment (martojdk) [inline src/main/java/com/person/Person.java:60]: leftovers from merge

File: src/main/java/com/service/CreatePersonService.java [hunk 6]
```diff
@@ -154,4 +165,73 @@ public EStatus createPersonStatus() {
...
+                    if (firstAnswer >= 1 && firstAnswer <= 10) {
+                        System.out.format(NEXT_ANSWER, firstAnswer);
+                        System.out.println();
+                        System.out.println(SECOND_QUESTION);
+                        try {
+                            secondAnswer = getUserInputInt();
+                            int answerValue = 0;
+                            switch (secondAnswer) {
+                                case 1:
+                                    answerValue = 10;
```
Comment (martojdk) [inline src/main/java/com/service/CreatePersonService.java:190]: Why do you write the values in the txt file, when you hardcode it anyway

File: src/main/java/com/service/Test.java (hunk 2 above)
Comment (martojdk) [reply src/main/java/com/service/Test.java:55]: Probably this answers your question

Comment (kostana1) [reply src/main/java/com/service/Test.java:55]: You think the counter should be == 5 ?

Comment (kostana1) [reply src/main/java/com/service/Test.java:55]: Come on, man. Spill the beans

Comment (martojdk) [reply src/main/java/com/service/Test.java:55]: Move the code from line 55 above line 52

Comment (kostana1) [reply src/main/java/com/service/Test.java:55]: A magic just happen. Big thank you

**Pull request closed without merging**